Neos.Neos/Documentation/ExtendingNeos/CustomEditPreviewMode.rst

```diff
@@ -30,6 +30,10 @@ Edit/preview modes are added to the Neos-Backend via *Settings.yaml*.
             fusionRenderingPath: 'print'
             # show after the existing modes
             position: 200
+            # sets the width of iframe (new UI)
```

Comment:
# sets the width of the iframe (React UI only)

Reply:
Done